spikeinterface/postprocessing/principal_component.py

```diff
@@ -229,6 +229,11 @@ def _run(self, n_jobs=1, progress_bar=False):
         we = self.waveform_extractor
         num_chans = we.get_num_channels()
 
+        # update job_kwargs with global ones
+        job_kwargs = fix_job_kwargs(job_kwargs)
+        n_jobs = job_kwargs['n_jobs'] if 'n_jobs' in job_kwargs else 1
```

Comment:
With the fix n_jobs is always in job_kwargs no ?
I would do `n_jobs = job_kwargs['n_jobs']`